src/api/modules/query.js

```diff
@@ -49,14 +49,14 @@ export const deleteOne = (model) => (req, res, next) => {
 }
 
 export const getOne = (model) => (req, res, next) => {
-  return controllers.getOne(req.docToUpdate)
+  return controllers.getOne(req.docFromId)
```

Comment:
I have confirmed this fixes fetching `GET /api/{resourceName}/:id` to return data in the response body. This also fixes the issue in  the course video `Config & Tests` > `Writing Tests` (5min 25s mark) where Scott doesn't see anything returning from logging `result.body`